toolchain/driver/testdata/error_codegen_output_file.carbon

```diff
@@ -0,0 +1,8 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// RUN: %{not} %{carbon} dump objcode --target_triple=x86_687-unknown-linux-gnu %s | %{FileCheck-strict}
+// CHECK:STDERR: ERROR: {{.*}}
```

Comment:
Can we check the error message here and in the other test?

Reply:
The reason I am not checking the error message is, we get the error message from llvm and if it's changed on their side, these tests will fail. 
If we want to check the entire error message, instead of using llvm's error message we can use our own generic error message. However, in this case the error message may not be meaningful.

Reply:
I'm expecting the error here to be the "Must provide an output file." error from driver.cpp, not something from LLVM.

Reply:
I am sorry for misunderstanding this. Added the error message check for this one.